ebuild-maintenance/removal/text.xml

```diff
@@ -113,16 +113,13 @@ In order to remove a virtual package, follow the following procedure:
     Wait the time appropriate for the last rites.
   </li>
   <li>
-    Update all ebuilds not to reference the virtual. Since there is
-    no urgent need to remove the virtual from user systems
-    and the resulting rebuilds would be unnecessary, do not bump ebuilds
-    when replacing the dependency.
+    Update all ebuilds not to reference the virtual with a new revision.
   </li>
   <li>
-    Remove the package directly
+    Remove the package directly.
   </li>
   <li>
-    Perform the post-removal cleanup, as with regular packages
+    Perform the post-removal cleanup, as with regular packages.
```

Comment:
No full stops there, see the [style guide](https://devmanual.gentoo.org/appendices/devbook-guide/index.html#external-coding-style).

Reply:
OK.